scripts/setup/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "nightly-2022-05-19"
+channel = "nightly-2022-06-30"
```

Comment:
I think this should be a separate PR and expect it to be merged first, to build the build-tool image.

Reply:
Only update `scripts/setup/rust-toolchain.toml`?

Reply:
https://databend.rs/doc/faq/routine-maintenance

Reply:
This workflow seems broken.

`build-tools` also requires changes in `rust-toolchain.toml`.

Reply:
@everpcpc for help